packages/api/src/webhooks/cannon.ts

```diff
@@ -222,6 +247,60 @@ export default class WebhookCannon {
     );
   }
 
+  async notifyFailedWebhook(
+    trigger: messages.WebhookTrigger,
+    params?: RequestInitWithTimeout,
+    err?: any
+  ) {
+    if (!trigger.user.emailValid) {
+      console.error(
+        `Webhook Cannon| User email is not valid, id: ${trigger.id}, streamId: ${trigger.stream?.id}`
+      );
+      return;
+    }
+    if (!(this.supportAddr && this.sendgridTemplateId && this.sendgridApiKey)) {
+      console.error(
+        `Webhook Cannon| Unable to send notification email to user, id: ${trigger.id}, streamId: ${trigger.stream?.id}`
+      );
+      console.error(
+        `Sending emails requires supportAddr, sendgridTemplateId, and sendgridApiKey`
+      );
+      return;
+    }
+
+    let signature_header = "";
+    if (params.headers[SIGNATURE_HEADER]) {
+      signature_header = `-H  "${SIGNATURE_HEADER}: ${params.headers["Livepeer-Signature"]}"`;
+    }
+
+    let payload = params.body;
+
+    await sendgridEmail({
+      email: trigger.user.email,
+      supportAddr: this.supportAddr,
+      sendgridTemplateId: this.sendgridTemplateId,
+      sendgridApiKey: this.sendgridApiKey,
+      subject: "Your webhook has been disabled",
+      preheader: "Failure notification",
+      buttonText: "Manage your webhooks",
+      buttonUrl: `https://${this.frontendDomain}/dashboard/developers/webhooks`,
+      unsubscribe: `https://${this.frontendDomain}/contact`,
+      text: [
+        `Your webhook ${trigger.webhook.url} failed to receive our payload in the last 24 hours`,
+        //`<code>${payload}</code>`,
+        //`This is the error we are receiving:`,
+        //`${err}`,
+        //`We disabled your webhook, please check your configuration and try again.`,
+        //`If you want to try yourself the call we are making, here is a curl command for that:`,
+        //`<code>curl -X POST -H "Content-Type: application/json" -H "user-agent: livepeer.com" ${signature_header} -d '${payload}' ${trigger.webhook.url}</code>`,
```

Comment:
followup from other comment
```suggestion
        //`<code>curl -X POST -H "Content-Type: application/json" -H "user-agent: livepeer.com" ${signatureHeader} -d '${payload}' ${trigger.webhook.url}</code>`,
```